lib/api/api.js

```diff
@@ -55,6 +55,11 @@ auth.setHandler(vault);
 /* eslint-disable no-param-reassign */
 const api = {
     callApiMethod(apiMethod, request, response, log, callback) {
+        log.addDefaultFields({
+            service: 's3',
+            action: apiMethod,
```

Comment:
unfortunately apiMethod != s3 action. For example, when the S3 action is [GetObject](https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html), the apiMethod is objectGet.

Can't we take advantage of the actionMap object from apiUtils/authorization/permissionChecks.js ?

Reply:
Ah yeah, we reverse the action names in the apiMethod. The actionMap is a good idea. Thanks!